gtk/OptionsDialog.cc

```diff
@@ -334,14 +334,13 @@ void TorrentFileChooserDialog::onOpenDialogResponse(int response, Glib::RefPtr<S
 {
     if (response == TR_GTK_RESPONSE_TYPE(ACCEPT))
     {
-        /* remember this folder the next time we use this dialog */
-        gtr_pref_string_set(TR_KEY_open_dialog_dir, IF_GTKMM4(get_current_folder, get_current_folder_file)()->get_path());
-
         bool const do_start = gtr_pref_flag_get(TR_KEY_start_added_torrents);
         bool const do_prompt = get_choice(std::string(ShowOptionsDialogChoice)) == "true";
         bool const do_notify = false;
 
-#if GTKMM_CHECK_VERSION(4, 0, 0)
+#if GTKMM_CHECK_VERSION(4, 10, 0)
```

Comment:
Another thing I've noticed when looking through GTKMM4 APIs
https://gnome.pages.gitlab.gnome.org/gtkmm/classGtk_1_1FileChooser.html#add83206926d71af1f50bb89bdb19b652

Reply:
Okay, I'll update this, thanks!

Reply:
In the end, `IF_GTKMM4` can be used here, so I used it instead.